indicators/NO_URGR_001-006/R/NO_URGR_001-006.qmd

```diff
@@ -0,0 +1,2547 @@
+```{r}
+#| eval: false
+#| code-summary: "Calculate SOAA impervious and non-impervious areas"
+
+fylke_soaa_imper_area <- fylke_soaa_imper %>%
+                          mutate(imper_area_mean = soaa_area_m2 * mean/100,
+                          non_imper_area_mean = soaa_area_m2 - imper_area_mean)
+```
+
+Finally, we can produce some summary tables for URGR_31. Note that two tables are produced: (i) one per fylke; (ii) one per region.
+
+```{r}
+#| eval: false
+#| code-summary: "Create summary tables"
+
+# Calculate summary table per fylke
+URGR_006_tab_fylke <- fylke_soaa_imper_area %>%
+                        select(fylkesnumm, fylkesnavn, clc18_kode, soaa_area_m2,
+                                imper_area_mean, non_imper_area_mean) %>%
+                        group_by(fylkesnumm, fylkesnavn) %>%
+                        summarise(total_imper_area_m2 = sum(imper_area_mean),
+                                  total_non_imper_area_m2 = sum(non_imper_area_mean),
+                                  total_soaa_area_m2 = sum(soaa_area_m2))
+
+# Calculate summary table per region
+URGR_006_tab_fylke$fylkesnumm <- as.numeric(URGR_006_tab_fylke$fylkesnumm)
+URGR_006_tab_region <- URGR_006_tab_fylke %>%
+                          left_join(., 
+                                    admin_bd[, c("fylke_nr", "region_name")], 
+                                    join_by(fylkesnumm == fylke_nr)) %>%
+                          group_by(region_name) %>%
+                          summarise(total_imper_area_m2 = sum(total_imper_area_m2),
+                                    total_non_imper_area_m2 = sum(total_non_imper_area_m2),
+                                    total_cty_grp_area_m2 = sum(total_soaa_area_m2))
+```
+
+
+These table have been exported to the internal repository and will be used in the Results section to create an overall summary table comparing all URGR indicators.
+
+
+
+
+
+
+
+
+<!--# 
+
+Use this header for documenting the analyses. Put code in separate code chunks, and annotate the code in between using normal text (i.e. between the chunks, and try to avoid too many hashed out comments inside the code chunks). Add subheaders as needed. 
+
+Code folding is activated, meaning the code will be hidden by default in the html (one can click to expand it).
+
+Caching is also activated (from the top YAML), meaning that rendering to html will be quicker the second time you do it. This will create a folder inside you project folder (called INDICATORID_cache). Sometimes caching created problems because some operations are not rerun when they should be rerun. Try deleting the cash folder and try again.
+
+-->
+
+## 10. Results
+
+### National results
+```{r}
+#| eval: true
+#| code-summary: "Code to create a comparative URGR table"
+
+# Load the results files
+path_res <- "/data/P-Prosjekter2/412413_2023_no_egd/Results/URGR/"
```

Comment:
were these produced with the code above? If yes, why do they need to be imported again? Or are these tables produce otherwise?